src/main/resources/format.yaml

```diff
@@ -3,6 +3,10 @@ repository: "xstefank/test-repo"
 statusUrl: "https://github.com/xstefank/test-repo/blob/master/.github/pull_request_template.md"
 
 format:
+  skippingPatterns:
+    titlePattern: "((.*)?NO JIRA REQUIRED(.*)?)|([^do not|don't]\\sskip.*template.*check(.*)?)|([^do not|don't]\\sbypass.*template.*check(.*)?)"
```

Comment:
generally a really good PR :) one last minor comment, just a matter of taste, can you rename skippingPatterns to skipPatterns, and titlePattern -> title (and other two the same) please?